.github/actions/placeholders-to-secrets/action.yaml

```diff
@@ -0,0 +1,43 @@
+name: Replace placeholders with secrets in build artifacts
+description: Replace placeholder variables in static site build artifacts
+
+inputs:
+  app-artifact-path:
+    description: Path to the app artifact directory
+    required: true
+
+  static-site-transforms:
```

Comment:
Should this be something more like `placeholder-transforms`?

Reply:
@robdmoore done